kpops/cli/pipeline_config.py

```diff
@@ -22,6 +22,18 @@ class TopicNameConfig(BaseSettings):
     )
 
 
+class SchemaRegistryConfig(BaseSettings):
+    enabled: bool = Field(
+        default=False,
+        description="If the Schema Registry handler should be initialized.",
+    )
+    url: AnyHttpUrl = Field(
+        default=parse_obj_as(AnyHttpUrl, "http://localhost:8081"),
```

Comment:
is it the recommended way to use `parse_obj_as`?

Reply:
Yeah, it is. I have added a comment and point out to this issue:  https://github.com/pydantic/pydantic/issues/1106